src/vivarium/framework/results/observer.py

```diff
@@ -51,6 +53,10 @@ def get_configuration_name(self) -> str:
         """Return the name of a concrete observer for use in the configuration"""
         return self.name.split("_observer")[0]
 
+    def get_configuration(self, builder: Builder) -> LayeredConfigTree:
+        config = builder.configuration["stratification"][self.get_configuration_name()]
```

Comment:
Why don't we do this?
```python
return builder.configuration.get_tree(["stratification", self.get_configuration_name()])
```

Reply:
Mypy

Reply:
It doesnt like it if I return a dict[str, Any] since the super class returns LCT or None. And if I ignore overriding the super class error it interprets the return you specific as just Any which we know is not really correct

Reply:
Oh I misread this comment. This works and I have updated it.